### PR TITLE
Missing configurations to enable SNAPSHOT release pipelines

### PR DESCRIPTION
**Pull request by manusa**
## Description
- Fix plugin management settings
- Include require settings to allow the creation of a SNAPSHOT release pipeline

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] My code follows the style guidelines of this project
 - [x] I Keep It Small and Simple: The smaller the PR is, the easier it is to review and have it merged
 - [x] I use [conventional commits](https://www.conventionalcommits.org/) in my commit messages
 - [x] I have performed a self-review of my code
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Review by rohanKanojia — APPROVED** [review]

File: jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/helper/VolumeBindingUtil.java [hunk 1]
```diff
@@ -100,7 +100,7 @@ private VolumeBindingUtil() { }
      * Resolves relative paths in the supplied {@code bindingString}, and returns a binding string that has relative
      * paths replaced with absolute paths.  If the supplied {@code bindingString} does not contain a relative path, it
      * is returned unmodified.
-     * <h3>Discussion:</h3>
+     * <h4>Discussion:</h4>
```
Comment (rohanKanojia) [inline jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/helper/VolumeBindingUtil.java:103]: /home/circleci/project/jkube-kit/build/service/docker/target/generated-sources/delombok/org/eclipse/jkube/kit/build/service/docker/helper/VolumeBindingUtil.java:103: error: header used out of sequence: <H4>
[ERROR]      * <h4>Discussion:</h4>

Comment (manusa) [reply jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/helper/VolumeBindingUtil.java:103]:
```
[ERROR] /home/circleci/project/jkube-kit/build/service/docker/target/generated-sources/delombok/org/eclipse/jkube/kit/build/service/docker/helper/VolumeBindingUtil.java:103: error: header used out of sequence: <H4>
[ERROR]      * <h4>Discussion:</h4>
[ERROR]        ^
```

OMG different Java versions are complaining of different settings. Locally with Java17 had the same error with H3. 
Not a good idea to add titles to Javadocs removing them...

Comment (rohanKanojia) [reply jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/helper/VolumeBindingUtil.java:103]: Sounds good to remove it.

**Comment by manusa** [comment]
```
maven/mavencentral/org.sonarsource.scanner.maven/sonar-maven-plugin/3.9.1.2184, LGPL-3.0-or-later, approved, #6944
```

**Pull request merged**